arch/arm/boot/dts/overlays/revpi-core-se-2022-overlay.dts

```diff
@@ -25,6 +25,8 @@
 
 	fragment@8 {
 		target = <&spi0>;
+		status = "disabled";
```

Comment:
I don't remember why, but I think we left the spi0 intentionally enabled. Give me until next week, I will try to remember why.

Reply:
Did you already had time to think about this?

Reply:
Yeah, I can think of anything else then that picontrol would have failed otherwise. So this should be ok

Reply:
So, this is good to merge then?